sw/device/tests/sram_ctrl_execution_test.c

```diff
@@ -75,11 +75,9 @@ bool test_main(void) {
 
   // Unlock the entire address space for RWX so that we can run this test with
   // both rom and test_rom.
-  CSR_SET_BITS(CSR_REG_PMPADDR15, 0x7fffffff);
-  CSR_SET_BITS(CSR_REG_PMPCFG3, kEpmpPermLockedReadWriteExecute << 24);
-  CSR_WRITE(CSR_REG_PMPCFG2, 0);
-  CSR_WRITE(CSR_REG_PMPCFG1, 0);
-  CSR_WRITE(CSR_REG_PMPCFG0, 0);
+  CSR_SET_BITS(CSR_REG_PMPADDR7, 0x7fffffff);
+  CSR_SET_BITS(CSR_REG_PMPCFG1,
```

Comment:
For pmpcfg1, I recommend do a CSR read to confirm that none of the regions are being configured (i.e. have content 0), and do a `CSR_WRITE`.